src/Core/Models/Api/Request/CipherRequestModel.cs

```diff
@@ -204,6 +204,7 @@ public class CipherBulkDeleteRequestModel
     {
         [Required]
         public IEnumerable<string> Ids { get; set; }
+        public string OrganizationId { get; set; }
```

Comment:
Should this not be required?

Reply:
It isn't always used, at least right now. A delete request that comes from a personal vault isn't going to have an OrganizationId attached.

Reply:
I see. Then you need to inline validate that it is there in the admin endpoints.

Reply:
Done!